packages/amplify-python-template-provider/amplify-plugin.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "functionTemplate",
+  "type": "util",
+  "commands": [],
+  "eventHandlers": [],
+  "functionTemplate": {
+    "conditions": {
+      "provider": "awscloudformation",
+      "service": "Lambda",
+      "runtime": "python"
+    },
+    "templates": [
+      {
+        "name": "Hello World",
+        "value": "helloworld"
```

Comment:
we should standardize it to hello-world like in the template directory, easier to locate and could be a good convention across different language packages